docs/Guides/zopen.md

```diff
@@ -0,0 +1,119 @@
+# zopen framework 
+
+The following is a description of the zopen tools provided in the meta repo. To obtain the latest tools, you can clone and set up the environment as follows:
+
+```
+git clone git@github.com:ZOSOpenTools/meta.git
+cd meta
+. ./.env
+```
+
+## If you are using z/OS Open Tools
+
+### zopen init
+To initialize the zopen installation directory to a location other than the default ($HOME/zopen), you can run the command zopen init and specify the desired directory. This will configure the directory for future use. Subsequently, tools like zopen download will download and install files to this specified directory."
+
+### zopen cacert
+
+To update the cacert.pem file, you can use `zopen update-cacert`. This will download the latest cacert.pem file https://curl.se/docs/caextract.html. This cacert.pem file is then used by other tools such as `zopen download` and `zopen build`.
+
+### zopen download
+
+To download and install the latest software packages, you can use `zopen download`. By default it will download all of the binaries hosted on ZOSOpenTools.
+
+It is recommended that you generate a [github personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+Then set `export ZOPEN_GIT_OAUTH_TOKEN=<yourapitoken>`
+
+To list the available packages, specify the `--list` option as follows:
+```
+zopen download --list
+```
+
+To download and install specfic packages, you can specify the packages as a comma seperated list as follows:
+```
+zopen download make,gzip
+```
+
+This will download it to the current working directory. To change the destination directory, you can specify the `-d` option as follows:
+
+```
+zopen download make -d $HOME/zopen/prod
+```
+
+## If you are contributing to or developing z/OS Open Tools
+
+### zopen build
+
+To build a software package, you can use `zopen build`.
+
+`zopen build` requires the files scripts in the project's root directory:
+- `buildenv`, which `zopen build` will automatically source.  If you would like to source another file, you can specify it via the `-e` option as in: `zopen build -e mybuildenv`
+
+The `buildenv` file _must_ set the following environment variables:
+- `ZOPEN_TYPE`: one of _TARBALL_ or _GIT_ indicating where the source should be pulled from (a source tarball or git repository)
+- `ZOPEN_URL`: the URL where the source should be pulled from, including the `package.git` or `package-V.R.M.tar.gz` extension
+- `ZOPEN_DEPS`: a space-separated list of all software dependencies this package has. These packages will automatically be downloaded if they are not present in your $HOME/zopen/prod or $HOME/zopen/boot directories.
+
+To help guage the build quality of the port, a `zopen_check_results()` function needs to be provided inside the buildenv. This function should process
```

Comment:
gauge not guage 
should we make the zopen_check_results a _must_ ? The code can't be used by anyone until it passes it's tests.

Reply:
I'll double check the code if we skip it if not defined or error out, but I've updated the typo